src/screens/WhotoFollow/index.js

```diff
@@ -64,7 +64,7 @@ const WhotoFollow = () => {
   const create = useClientGetstream();
 
   const navigation = useNavigation();
-  const {bottom} = useSafeAreaInsets();
+  const {bottom, top} = useSafeAreaInsets();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `{bottom}` to `{bottom, top}` in the `useSafeAreaInsets()` hook seems fine. It's a good practice to destructure the values you need directly from the hook. However, ensure that the `top` value is being used correctly throughout the rest of your code.
```diff
-  const {bottom} = useSafeAreaInsets();
+  const {bottom, top} = useSafeAreaInsets();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->